### ISSUE TITLE
Using same ticker BLC as Blakecoin

### ISSUE THREAD

**Issue by BlueDragon747** [BLCdeveloper/cryptobullcoin#1]
Why are you using same ticker as Blakecoin - BLC did you guys not even check before release or what?

really you should have picked a ticker that is not used expect all sorts of confusion Blakecoin has been using BLC since 2013 and will continue to do so

**Comment by christongsing** [comment]
yes you might use CB OR CBC so they not usuallly use by other crypto coins

**Comment by hotlocco** [comment]
It's been more than six months, still no fix.

**Comment by BlueDragon747** [comment]
its vol on c-cex looks low probs gonna be delisted people behind this coin retired or are working on next coin hence why they never fixed it or even responded
